Test/git-issues/git-issue-2441.dfy

```diff
@@ -0,0 +1,7 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
```

Comment:
Does this work for every target language now? Can we use `%testDafnyForEachCompiler`?

Reply:
No, python fail, but the fix is in a different PR. Go fails because of an unrelated issue.